app/models/user.rb

```diff
@@ -0,0 +1,11 @@
+class User < ApplicationRecord
+  class User < ApplicationRecord
+    has_many :posts, foreign_key: 'author_id'
+    has_many :likes, foreign_key: 'author_id'
+    has_many :comments, foreign_key: 'author_id'
+
+    def recent_three_posts
+      posts.last(3)
+    end
```

Comment:
- [ ] We want to fetch the most recent posts. This means if we have 5 posts we want to get the fifth, fourth, and third posts - in that order. But this `#last` method means we are fetching the posts in this order: third, fourth, and fifth.
  
  I would recommend you change this to something like this: `posts.order('created_at Desc').limit(3)`

  ![last-3](https://user-images.githubusercontent.com/81468675/190429222-c242e1af-b6bf-424e-b9d3-d3acb4f9b9e0.jpg)